openmc/data/kalbach_mann.py

```diff
@@ -13,396 +13,154 @@
-                             projectile,
-                             target,
-                             compound,
-                             emitted,
-                             residual):
-    """Calculate the Kalbach slope for projectiles other than photons
-    as defined in ENDF-6 manual BNL-203218-2018-INRE, Revision 215,
-    File 6 description for LAW=1 and LANG=2.
-
-    The entrance and emission channel energies are not calculated with
-    the AWR number, but approximated with the number of mass instead.
-
-    Parameters
-    ----------
-    energy_projectile: float
-        Energy of the projectile in the laboratory system in eV
-    energy_emitted: float
-        Energy of the emitted particle in the center of mass system in eV
-    projectile: AtomicRepresentation
-        Atomic representation of the projectile
-    target: AtomicRepresentation
-        Atomic representation of the target
-    compound: AtomicRepresentation
-        Atomic representation of the compound
-    emitted: AtomicRepresentation
-        Atomic representation of the emitted particle
-    residual: AtomicRepresentation
-        Atomic representation of the residual nucleus
-
-    Returns
-    -------
-    slope: float
-        Kalbach-Mann slope
-
-    """
-    epsilon_a = _return_entrance_channel_energy(
-        energy_projectile,
-        target.a,
-        projectile.a
-    ) / EV_PER_MEV
-    epsilon_b = _return_emission_channel_energy(
-        energy_emitted,
-        residual.a,
-        emitted.a
-    ) / EV_PER_MEV
-
-    s_a = _calculate_separation_energy(compound, target, projectile)
-    s_b = _calculate_separation_energy(compound, residual, emitted)
-
-    e_a = epsilon_a + s_a
-    e_b = epsilon_b + s_b
-
-    r_1 = min(e_a, _ET1)
-    r_3 = min(e_a, _ET3)
-
-    x_1 = r_1 * e_b / e_a
-    x_3 = r_3 * e_b / e_a
-
-    slope = _C1 * x_1 \
-        + _C2 * x_1**3 \
-        + _C3 * projectile.M * emitted.m * x_3**4
-
-    return slope
+    # Determine A, Z, and N for compound and nucleus
+    A_c = compound.a
+    Z_c = compound.z
+    N_c = compound.n
+    A_a = nucleus.a
+    Z_a = nucleus.z
+    N_a = nucleus.n
+
+    # Determine breakup energy of incident particle (ENDF-6 Formats Manual,
+    # Appendix H, Table 3)
+    za_to_breaking_energy = {
+        1: 0.0,
+        1001: 0.0,
+        1002: 2.224566,
+        1003: 8.481798,
+        2003: 7.718043,
+        2004: 28.29566
+    }
+    I_b = za_to_breaking_energy[particle.za]
+
+    # Eq. 4 in in doi:10.1103/PhysRevC.37.2350 or ENDF-6 Formats Manual section
+    # 6.2.3.2
+    return (
+        15.68 * (A_c - A_a) -
+        28.07 * ((N_c - Z_c)**2 / A_c - (N_a - Z_a)**2 / A_a) -
+        18.56 * (A_c**(2./3.) - A_a**(2./3.)) +
+        33.22 * ((N_c - Z_c)**2 / A_c**(4./3.) - (N_a - Z_a)**2 / A_a**(4./3.)) -
+        0.717 * (Z_c**2 / A_c**(1./3.) - Z_a**2 / A_a**(1./3.)) +
+        1.211 * (Z_c**2 / A_c - Z_a**2 / A_a) -
+        I_b
```

Comment:
```suggestion
        I_a
```
For consistency with the above modification